package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "blackbaud-skyux-spa-rrrrr-app-name",
+  "version": "0.0.1",
+  "description": "Single-page-application for rrrrr-app-name",
+  "main": "index.js",
+  "scripts": {
+    "skyux": "skyux",
+    "build": "skyux build",
+    "test": "skyux test",
+    "test:watch": "skyux watch",
+    "start": "skyux serve",
+    "e2e": "skyux e2e",
+    "lint": "skyux lint"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
Thanks for doing the PR @michael-tims!  When the `@skyux-sdk/cli` runs `skyux new`, it will replace any instance of `latest` in this file with the actual latest version.  I would make that change.  For comparison, you can see what the default template looks like at https://github.com/blackbaud/skyux-sdk-template/blob/master/package.json